### PR TITLE
[chore] Consolidate genai.request.seed

### PR DESCRIPTION
**Pull request by gyliu513**
Fixes #

This is a follow up for https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3081/files#r1884746832

/cc @lmolkova @lzchen 

## Changes

Please provide a brief description of the changes here.

Note: if the PR is touching an area that is not listed in the [existing areas](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/README.md), or the area does not have sufficient [domain experts coverage](https://github.com/open-telemetry/semantic-conventions/blob/main/.github/CODEOWNERS), the PR might be tagged as [experts needed](https://github.com/open-telemetry/semantic-conventions/labels/experts%20needed) and move slowly until experts are identified.

## Merge requirement checklist

* [ ] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [ ] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

**Comment by lmolkova** [comment]
@gyliu513 thanks for the contribution!

The markdown tables are auto-generated from yaml definitions - please check https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#1-modify-the-yaml-model for more details. 

Also, we don't remove attributes, we deprecate them - you can find some examples in the https://github.com/open-telemetry/semantic-conventions/blob/main/model/gen-ai/deprecated/registry-deprecated.yaml

We'll also need a changelog record for it - you can find more details on how to create one in https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#4-changelog.

File: .chloggen/1683.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
```
Comment (lmolkova) [inline .chloggen/1683.yaml:7]:
```suggestion
change_type: breaking
```

File: .chloggen/1683.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: gen_ai
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Consolidate genai.request.seed
```
Comment (lmolkova) [inline .chloggen/1683.yaml:13]:
```suggestion
note: Rename `gen_ai.openai.request.seed` to `gen_ai.request.seed` and use it on general GenAI conventions.
```

File: model/gen-ai/spans.yaml [hunk 3]
```diff
@@ -26,6 +26,8 @@ groups:
         requirement_level: recommended
       - ref: gen_ai.request.presence_penalty
         requirement_level: recommended
+      - ref: gen_ai.request.seed
+        requirement_level: recommended
```
Comment (lmolkova) [inline model/gen-ai/spans.yaml:30]:
```suggestion
        requirement_level:
          conditionally_required: if appliable and if the request includes a seed
```

**Review by lmolkova — APPROVED** [review]
Thanks! 

Left a few minor comments. 

Please also update the https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml - add the following under version 1.29.0

```yaml
        # https://github.com/open-telemetry/semantic-conventions/pull/1683
        - rename_attributes:
            attribute_map:
              gen_ai.openai.request.seed: gen_ai.request.seed
```

File: .chloggen/1683.yaml (hunk 1 above)
Comment (gyliu513) [reply .chloggen/1683.yaml:7]: Done

File: .chloggen/1683.yaml (hunk 2 above)
Comment (gyliu513) [reply .chloggen/1683.yaml:13]: Done

**Review by gyliu513 — COMMENTED** [review]
Thanks @lmolkova , all comments are addressed.

File: schema-next.yaml [hunk 4]
```diff
@@ -12,6 +12,7 @@ versions:
         # https://github.com/open-telemetry/semantic-conventions/pull/1383
         - rename_attributes:
             attribute_map:
+              gen_ai.openai.request.seed: gen_ai.request.seed
```
Comment (lmolkova) [inline schema-next.yaml:15]: sorry, I mislead you with my prev comment, please add 

```yaml
    all:
      changes:
        - rename_attributes:
            attribute_map:
              # https://github.com/open-telemetry/semantic-conventions/pull/1683
              gen_ai.openai.request.seed: gen_ai.request.seed
```

under `next:` (the current version, 1.29.0 is already released)

**Review by lmolkova — APPROVED** [review]

File: schema-next.yaml (hunk 4 above)
Comment (gyliu513) [reply schema-next.yaml]: Done, I should always update next, thanks @lmolkova !

**Review by lzchen — APPROVED** [review]
Awesome, thanks for this!

**Comment by lmolkova** [comment]
It seems @xrmx's PR got in faster #1710. Sorry about this, but thanks a lot @gyliu513 !

**Pull request closed without merging**